app/models/match_decisions/homeless_set_aside/set_asides_confirm_hsa_accepts_client_decline_dnd_staff.rb

```diff
@@ -83,9 +83,9 @@ def decline_confirmed
         match.rejected!
       end
 
-      def canceled
+      def canceled contact
         Notifications::MatchCanceled.create_for_match! match
-        match.canceled!
+        match.canceled! contact&.id
```

Comment:
`ClientOpportunityMatch#canceled!` takes a named parameter, so this won't work as written.